kgforge/specializations/models/rdf/rdf_model_service_from_store.py

```diff
@@ -91,22 +91,24 @@ def _build_shapes_map(self) -> Dict:
                     }}
                 }}
             }} ORDER BY ?type"""
+
         # make sure to get all types
         limit = 100
         offset = 0
         count = limit
-        class_to_shapes = dict()
-        shape_resource = dict()
+        class_being_shaped_id_to_shape_uri = dict()
+        schema_to_resource: Dict[URIRef, URIRef] = dict()
+
         while count == limit:
             resources = self.context_store.sparql(query, debug=False, limit=limit, offset=offset)
             for r in resources:
                 shape_uri = URIRef(r.shape)
-                class_to_shapes[r.type] = shape_uri
-                shape_resource[shape_uri] = URIRef(r.resource_id)
+                class_being_shaped_id_to_shape_uri[r.type] = shape_uri
+                schema_to_resource[shape_uri] = URIRef(r.resource_id)
```

Comment:
Actually it is shape to resource and schema to resource. To avoid confusion with what we call schema elsewhere

Reply:
Yes I wasn't paying much attention to my usage of schema/shape but i can use shape exclusively